src/test/java/org/opensearch/knn/ODFERestTestCase.java

```diff
@@ -138,8 +138,8 @@ protected void wipeAllODFEIndices() throws IOException {
 
             for (Map<String, Object> index : parserList) {
                 String indexName = (String) index.get("index");
-                if (indexName != null && !".opendistro_security".equals(indexName)) {
-                    adminClient().performRequest(new Request("DELETE", "/" + indexName));
+                if (indexName != null && !".opendistro_security".equals(indexName) && !indexName.matches("knn-bwc-(.*)")) {
+                    client().performRequest(new Request("DELETE", "/" + indexName));
```

Comment:
This should be adminClient(). This caused an issue during CI.

Reply:
Updated